PoseEstimation-TFLiteSwift/PoseConfidenceMapDrawingView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  PoseConfidenceMapDrawingView.swift
+//  PoseEstimation-TFLiteSwift
+//
+//  Created by Doyoung Gwak on 2020/04/24.
+//  Copyright © 2020 Doyoung Gwak. All rights reserved.
+//
+
+import UIKit
+
+class PoseConfidenceMapDrawingView: UIView {
+    typealias CGLine = (from: CGPoint, to: CGPoint)
+    var outputChannelIndexes: [Int] = [0]
```

Comment:
Indexes (x)
Indice (o)

Reply:
`indexes` also correct on the meaning view, but I'll accept your suggestion in the technical view.

ref: https://english.stackexchange.com/a/3126